Libraries/Cordio/docs/Applications/Bootloader.md

```diff
@@ -0,0 +1,31 @@
+# Description
```

Comment:
These README files can stay in place. This isn't specifically a Cordio application.

Reply:
By these I mean the Bootloader files.